app/src/main/java/com/example/itacademyhw/UserViewHolder.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.itacademyhw
+
+import android.graphics.BitmapFactory
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.itacademyhw.room.model.UserEntity
+
+class UserViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    private val userItemView: TextView = itemView.findViewById(R.id.textView)
+    private val userImageView: ImageView = itemView.findViewById(R.id.userImageView)
+
+    fun bind(user: UserEntity) {
+        userItemView.text = "${user.firstName} ${user.lastName}"
+        if (user.photoName.isNotEmpty()) {
+            val file = itemView.context
+                .filesDir
+                .listFiles()
+                ?.filter { it.canRead() && it.isFile && it.name.endsWith(JPG_EXTENSION) }
```

Comment:
ты вычитваешь все файлы из какого то хранилища, можно найти по имени 
```suggestion
                ?.find { it.canRead() && it.isFile && it.name.endsWith(JPG_EXTENSION) && it.name.contains(user.photoName) }
```